### PR TITLE
Fix contract setup

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

**Comment by da-kami** [comment]
https://github.com/comit-network/hermes/issues/235

File: daemon/src/maker.rs [hunk 1]
```diff
@@ -149,7 +150,10 @@ async fn main() -> Result<()> {
         auth_password
     );
 
-    let oracle = schnorrsig::KeyPair::new(SECP256K1, &mut rand::thread_rng()); // TODO: Fetch oracle public key from oracle.
+    // TODO: Actually fetch it from Olivia
```
Comment (thomaseizinger) [inline daemon/src/maker.rs:153]: I think hard-coding is completely fine. If we fetch it from Olivia, we make ourselves vulnerable to MITM attacks.

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/maker.rs (hunk 1 above)
Comment (da-kami) [reply daemon/src/maker.rs:153]: ok, then I close #235 for now

**Pull request merged**